raco/expression/function.py

```diff
@@ -66,8 +116,19 @@ def typeof(self, scheme, state_scheme):
         return types.DOUBLE_TYPE
 
 
+class YEAR(UnaryFunction):
```

Comment:
What's the deal with special-casing this date function? Or is it just a test case for some new feature in this PR?